odoo/tools/template_inheritance.py

```diff
@@ -74,14 +74,17 @@ def locate_node(arch, spec):
     return None
 
 
-def apply_inheritance_specs(source, specs_tree, inherit_branding=False):
+def apply_inheritance_specs(source, specs_tree, inherit_branding=False, pre_locate=False):
```

Comment:
you could even do something like `pre_locate=lambda s: True` ([like here](https://github.com/odoo/odoo/blob/4057227def164f3c29c6d5546f1e8572dc0db573/odoo/models.py#L1048)). This way, you don't need the `if pre_locale`.

Reply:
done